.gitignore

```diff
@@ -1,2 +1,13 @@
 /node_modules/
 /target/
+
+# VSCode
```

Comment:
IMHO IDE-specific files should belong in your personal .gitignore file, not in project ignore files, see also http://stratus3d.com/blog/2018/06/03/stop-excluding-editor-temp-files-in-gitignore/

Reply:
sure, will remove. i usually just leave them since i find it helpful to get tips from a maintainer on what works well for a given ide, but doesn't have to be in code or anything